### ISSUE TITLE
select-apps: TaskCanceledException when attempting to select game(s)

### ISSUE THREAD

**Issue by ccl-io** [tpill90/steam-lancache-prefill#177]
## Description

Application crashing with a `TaskCanceledException` when the `select-apps` option is used to configure a collection of game(s) that would be prefilled to a LAN Cache.

## Reproduction

1. Login with this [account](https://steamcommunity.com/id/stux314/)
2. Execute with the `select-apps` option
3. See `TaskCanceledException` error

Described below are the host environments for both the LANCache and the supplementary Test Client. A copy of version `1.6.0` of the application was ran on both platforms to reproduce the issue.

- **LAN Cache Server**
  - **Device Parameters**: Custom 2U Rack Server
    - *CPU(s)*: 2x Intel Xeon Gold 6330
    - *RAM*: 128 GB DDR4 ECC
    - *Cache Storage*: 512 TB LUN
      - *SAN Uplink*: 192 Gbps [FC] via QLogic QLE2764-SR-CK HBA
    - *Network Uplink*: 100 GbE via Mellanox MCX623105AC
  - **OS**: Red Hat Enterprise Linux 7.9
  - **Terminal**: [Termius](https://termius.com/)
  - **Test Command**: `./SteamPrefill select-apps`
    - Note: *Login to Steam was successfully conducted prior to the test*
- **Test Client**      
  - **Device**: Microsoft Surface Laptop Studio
  - **OS**: Windows 10 Pro (21H1)
  - **Terminal**: Windows Terminal 
  - **Test Command**: `.\SteamPrefill.exe select-apps`
    - Note: *Login to Steam was successfully conducted prior to the test*

The [LANCache](https://lancache.net/) container was initialized on a newly built server running RHEL 7.9 [minimal] with a dedicated LUN allocation as cache storage.

The Cache was prefilled with Activision/Blizzard games through a related [Battle.net Prefiller](https://github.com/tpill90/battlenet-lancache-prefill) project. This application was then configured to log in to [my Steam Account](https://steamcommunity.com/id/stux314/) with the `select-apps` option before it crashed with the given exception:

```
[3:52:41 AM] Starting login!
[3:52:41 AM] Logged into Steam
[3:52:41 AM] Steam session initialization complete!
System.Threading.Tasks.TaskCanceledException: A task was canceled.
  at async Task SteamPrefill.Handlers.AppInfoHandler.BulkLoadAppInfosAsync(List<uint> appIds)                                              
  at void SteamPrefill.Handlers.AppInfoHandler.<>c__DisplayClass7_0.<<RetrieveAppMetadataAsync>b__0>d.MoveNext()                           
  at void Spectre.Console.Status.<>c__DisplayClass16_0.<<StartAsync>b__0>d.MoveNext() in Status.cs:79                                      
  at void Spectre.Console.Status.<>c__DisplayClass17_0`1.<<StartAsync>b__0>d.MoveNext() in Status.cs:120                                   
  at void Spectre.Console.Progress.<>c__DisplayClass28_0`1.<<StartAsync>b__0>d.MoveNext() in Progress.cs:133                               
  at async Task<T> Spectre.Console.Internal.DefaultExclusivityMode.RunAsync<T>(Func<Task<T>> func) in DefaultExclusivityMode.cs:40         
  at async Task<T> Spectre.Console.Progress.StartAsync<T>(Func<ProgressContext, Task<T>> action) in Progress.cs:116                        
  at async Task<T> Spectre.Console.Status.StartAsync<T>(string status, Func<StatusContext, Task<T>> func) in Status.cs:117                 
  at async Task Spectre.Console.Status.StartAsync(string status, Func<StatusContext, Task> action) in Status.cs:77                         
  at async Task SteamPrefill.Handlers.AppInfoHandler.RetrieveAppMetadataAsync(List<uint> appIds, bool loadDlcApps, bool loadRecentlyPlayed)
  at async Task<List<AppInfo>> SteamPrefill.SteamManager.GetAllAvailableGamesAsync()                                                       
  at async ValueTask SteamPrefill.CliCommands.SelectAppsCommand.ExecuteAsync(IConsole console)                                             
[3:52:52 AM] Disconnected from Steam!
```

Subsequent usages of the `select-apps` option produces the same exception on both the server and client prefiller.

## Screenshots
_Error on RHEL 7.9 LANCache Server:_
![image](https://user-images.githubusercontent.com/2673346/196015225-81d76cab-887f-4213-9155-b6cb44dab27c.png)
_Error on Windows 10 21H1 PC:_
![image](https://user-images.githubusercontent.com/2673346/196015214-434961c2-a6ac-40b6-b2d6-2280160427d0.png)

**Comment by tpill90** [comment]
![image](https://user-images.githubusercontent.com/766513/196251537-fe182c48-0514-4b29-9b62-a7bfa910c302.png)

![image](https://user-images.githubusercontent.com/766513/196251557-0be1508c-5f23-4fc2-bdc6-7067c29f6b71.png)

![image](https://user-images.githubusercontent.com/766513/196251589-46d82551-3dae-4d11-b3f4-27a381ce0b86.png)

**Comment by tpill90** [comment]
Released in v1.7.0 https://github.com/tpill90/steam-lancache-prefill/releases/tag/v1.7.0
